command/node_drain.go

```diff
@@ -128,14 +129,22 @@ func (c *NodeDrainCommand) Run(args []string) int {
 	flags.BoolVar(&keepIneligible, "keep-ineligible", false, "Do not update the nodes scheduling eligibility")
 	flags.BoolVar(&self, "self", false, "")
 	flags.BoolVar(&autoYes, "yes", false, "Automatic yes to prompts.")
+	flags.BoolVar(&monitor, "monitor", false, "Monitor drain status.")
 
 	if err := flags.Parse(args); err != nil {
 		return 1
 	}
 
+	// Check that enable or disable is not set with monitor
+	if monitor && enable || monitor && disable {
```

Comment:
`monitor && ( enable || disable )` ?

Reply:
Without the context, that looks kind of goofy.

Reply:
Thanks!